cmd/cloudshell_open/main.go

```diff
@@ -291,14 +292,24 @@ func run(opts runOpts) error {
 		}
 	}
 
+	end = logProgress(
+		fmt.Sprintf("Setting up %s in region %s (if it doesn't already exist)", highlight(artifactRegistry), highlight(region)),
+		fmt.Sprintf("Set up %s in region %s (if it doesn't already exist)", highlight(artifactRegistry), highlight(region)),
+		"Failed to setup artifact registry.")
+	err = createArtifactRegistry(project, region, artifactRegistry)
```

Comment:
Good point; I need to confirm if artifactregistry is a default API, but always good to make sure it's enabled (even in it's current state, the code doesn't check first then optionally enable, it just always enables). 

Don't think we also need to enable containerregistry any more (and it may not be available on newly created projects)